dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/event/RelationshipControllerTest.java

```diff
@@ -177,18 +176,11 @@ public void testGetRelationship()
         mockMvc.perform( get( ENDPOINT + "/" + REL_ID ) ).andExpect( status().isOk() );
     }
 
-    @Test( expected = NestedServletException.class )
-    public void testDeleteRelationshipNotPresent()
-        throws Exception
-    {
-        mockMvc.perform( delete( ENDPOINT + "/" + REL_ID ) ).andExpect( status().isConflict() );
-    }
-
```

Comment:
FYI: I removed this test since it now tested via actual controller interaction and would here assert `OK` which is confusing. The reason this test would need to assert `OK` when the actual result is `NOT_FOUND` is that the change to the HTTP response status is first made in an controller advice which is outside of this test setup.